TweetBook/Controllers/V1/PostsController.cs

```diff
@@ -30,16 +50,15 @@ public IActionResult Create([FromBody] CreatePostRequest postRequest)
 
             var post = new Post { Id = postRequest.Id };
 
-            if(string.IsNullOrEmpty(post.Id))
-                post.Id = Guid.NewGuid().ToString();
-            
-            _posts.Add(post);
+            if (post.Id != Guid.Empty)
```

Comment:
Here shouldn't we want to have a new Id only if we did not get one in the first place?